pkg/services/alerting/ticker_test.go

```diff
@@ -27,10 +27,8 @@ func TestTicker(t *testing.T) {
 	}
```

Comment:
I'm of the strong opinion that metrics should be tested.

A change in a metric should be considered a breaking change and thus we should be aware if anything changes: behaviour, name, etc.

Let me know if you need any help testing the metrics as I'm happy to suggest a quick and simple approach to assert on them.

Reply:
You'll also need a changelog entry for the new metrics 😄 

Something like:

```golang
- Alerting: New metrics for observing alert scheduling are introduced #XXX
    `grafana_alerting_ticker_last_consumed_tick_timestamp_seconds`
    `grafana_alerting_ticker_next_tick_timestamp_seconds`
    `grafana_alerting_ticker_interval_seconds`
```